Refit/HttpClientFactoryExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿#if NETSTANDARD2_0
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Refit
+{
+    public static class HttpClientFactoryExtensions
+    {
+        public static IHttpClientBuilder AddRefitClient<T>(this IServiceCollection services) where T : class
```

Comment:
I think we need an overload that can specify the refit settings. Can either be as another parameter or as an `Action<RefitSettings>` where the method news it up and passes it to the user for setting?